CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# CONTRIBUTING
+
+When contributing to this repository, please first discuss the change you wish to make via [issues](https://github.com/rigd-loxia/builder-generator/issues).
+
+Please note if you are working on a certain issue then make sure to stay active with development.
+
+## Git Commit, Branch, and PR Naming Conventions
+
+When you are working with git, please be sure to follow the conventions below on your pull requests, branches, and commits:
+
+```text
+PR: [#ISSUE ID] Title of the PR
+Branch: [ISSUE ID]-title-of-the-pr (shorter)
+Commit: [[ISSUE ID]] what was done
```

Comment:
Commits should be #[IssueId] so that it becomes a link. For example: `#8` becomes #8

Reply:
done